### ISSUE TITLE
Migrate example to Android embedding v2

### ISSUE THREAD

**Issue by nilsreichardt** [dnfield/flutter_svg#486]
## Description
The `example` of `flutter_svg` isn't migrated to Android embedding v2 yet.

```
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Warning
──────────────────────────────────────────────────────────────────────────────
Your Flutter application is created using an older version of the Android
embedding. It's being deprecated in favor of Android embedding v2. Follow the
steps at

https://flutter.dev/go/android-project-migration

to migrate your project.
```

### Steps to reproduce
1. Open example
2. Run `flutter pub get`

**Comment by dnfield** [comment]
Yeah this should be easy. I've been meaning to get to this, I'll push something up shortly.

**Comment by dnfield** [comment]
Fixed in 6c88819
